library/src/main/java/com/novoda/downloadmanager/lib/DownloadsUriProvider.java

```diff
@@ -59,14 +55,6 @@ private static DownloadsUriProvider newInstance() {
     }
 
     /**
-     * The content URI for accessing publicly accessible downloads (i.e., it requires no
-     * permissions to access this downloaded file)
-     */
-    public Uri getPubliclyAccessibleDownloadsUri() {
```

Comment:
yes be careful if something is public in a lin you have to really think about it. I think it was client facing :x: